### ISSUE TITLE
Release Flatcar Container Linux Alpha 3255.0.0, Beta 3227.1.0, Stable 3139.2.2, LTS-2021 2605.28.1 LTS-2022 3033.3.1

### ISSUE THREAD

**Issue by pothos** [flatcar/Flatcar#761]
## Release Flatcar Container Linux Alpha 3255.0.0, Beta 3227.1.0, Stable 3139.2.2, LTS-2021 2605.28.1, LTS-2022 3033.3.1
  
The releases are planned for 2nd June 2022

### Tasks
- [x] Go / No Go tracking [hackmd created](https://hackmd.io/6SCdWxODQzamuHMdmb3eIw) (@pothos)
- [x] All planned changes are merged & blocker bugs are fixed (@pothos, @sayanchowdhury )
- [x] Branch off main (only new major) (@dongsupark, @pothos, @sayanchowdhury )
- [x] Tag build branch (@dongsupark, @sayanchowdhury )
- [x] Generate manifest (@dongsupark, @sayanchowdhury )
- [x] Start Jenkins builds (@dongsupark, @pothos, @sayanchowdhury )
- [x] Test status captured in test status tracker (@pothos,, @sayanchowdhury)
- [x] Create SDK container after Jenkins is done building the flatcar-sdk tarball; upload container to ghcr.io
- [x] All tests are green (@sayanchowdhury, @pothos)
- [x] Archive TAP report, create Go / No Go doc ()
- [x] Release notes / announcements added to Go/No Go doc (@pothos)
- [x] QA: check diff for image file list, packages, and image size (@pothos)
- [x] Matrix Go/No Go meeting (inc. release notes review) (@sayanchowdhury)
- [x] Manifest release created on GitHub (@sayanchowdhury)
- [x] Update the SDK release metadata for new major Alpha releases with SDK builds (only new major) (@sayanchowdhury)
- [x] Images uploaded (@dongsupark)
- [x] Run the `os/release` job (@sayanchowdhury)
- [x] Publish EC2 Marketplace Images (@tormath1)
- [x] Azure offer updated & publishing started (@sayanchowdhury )
- [x] Azure Go Live ()
- [x] GCP offers updated (@sayanchowdhury)
- [x] Update payload signed/published (@dongsupark)
- [x] Symlink to "current" updated (@sayanchowdhury)
- [x] Website updated (@sayanchowdhury)
- [x] Update CloudFormation templates (@sayanchowdhury)
- [x] Release package published in Nebraska (@dongsupark)
- [x] Send announcement to Mailing Lists (@pothos)
- [x] Tweet out! (@miao0miao, @ahrkrak)

**Comment by pothos** [comment]
Links to the Jenkins builds:

- Alpha 3255.0.0 http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/5756/cldsv/ (was Alpha 3248.0.0 http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/5732/cldsv/)
- Beta 3227.1.0 http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/5687/cldsv/
- Stable 3139.2.2 http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/5686/cldsv/
- LTS-2022 3033.3.1 http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/5733/cldsv/
- LTS-2021 2605.28.1 http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/5760/cldsv/

**Comment by dongsupark** [comment]
Blocked on `Update payload signed/published` step, due to issues on Nebraska.

**Comment by sayanchowdhury** [comment]
The LTS-2021 2605.28.1 would need a separate issue, as the release was still building when we started the Alpha, Beta, Stable, and LTS-2022 was already ready.

The 2605.28.1 would be released on 06-06-2022

**Comment by tormath1** [comment]
Regarding the AWS marketplace update the following versions are currently excluded:
 * lts-2605.28.1 (release on monday), see :arrow_up: 
 * ARM64 lts-3033.3.1 (needs a new a offer)

**Comment by dongsupark** [comment]
Nebraska issues were all fixed.
Rolling updates started.
